scripts/bulk_index.pl

```diff
@@ -49,32 +49,40 @@ BEGIN
 
 $indexer->create_index;
 
+my $run_time = time;
 my $loop = IO::Async::Loop->new;
 my $timer = IO::Async::Timer::Periodic->new(
     first_interval => 0,
     interval       => 15,
     reschedule     => 'skip',
 
     on_tick => sub {
+        printf "Running after %d seconds\n", time - $run_time;
         my $start_users = time;
         say "indexing users" if $verbose;
-        $indexer->bulk_load('Bugzilla::User');
-        print "    ", time - $start_users, " seconds\n" if $verbose > 1;
+        my $users = $indexer->bulk_load('Bugzilla::User');
+        bulk_load_stats($start_users, $users) if $verbose > 1;
 
-        say "indexing bugs" if $verbose;
         my $start_bugs = time;
-        $indexer->bulk_load('Bugzilla::Bug');
-        print "    ", time - $start_bugs, " seconds\n" if $verbose > 1;
+        say "indexing bugs" if $verbose;
+        my $bugs = $indexer->bulk_load('Bugzilla::Bug');
+        bulk_load_stats($start_bugs, $bugs) if $verbose > 1;
 
-        say "indexing comments" if $verbose;
         my $start_comments = time;
-        $indexer->bulk_load('Bugzilla::Comment');
-        print "    ", time - $start_comments, " seconds\n" if $verbose > 1;
+        say "indexing comments" if $verbose;
+        my $comments = $indexer->bulk_load('Bugzilla::Comment');
+        bulk_load_stats($start_comments, $comments) if $verbose > 1;
 
         $loop->stop if $once;
+        $run_time = time;
```

Comment:
You should set this prior to $loop->stop, since otherwise the printf at +60 will never be accurate beyond a single run due to invalid start time. It's technically a minor point, but code outside this loop shouldn't have to worry about *whether* $run_time is valid, ever, and it costs little to make it right for all scenarios rather than just some.